src/appleseed-max-impl/appleseedlightmtl/appleseedlightmtl.cpp

```diff
@@ -43,7 +43,7 @@
 #include "renderer/api/material.h"
 #include "renderer/api/scene.h"
 #include "renderer/api/utility.h"
-#include "renderer/modeling/texture/sourceentity.h"
+#include "renderer/modeling/input/sourceentity.h"
```

Comment:
What we really need to do to make this official is introduce a new API header file in appleseed, `renderer/api/source.h`, that itself includes the appleseed's header files we want to expose (i.e. `renderer/modeling/input/sourceentity.h`, maybe others).